docs/get-started.md

```diff
@@ -71,36 +71,32 @@ Node.js 18 or later.
     1. Choose default answers to the rest of the questions, which will
        initialize your project folder with some sample code.
 
-    The `genkit init` command created two sample source files:
+    The `genkit init` command creates a sample source file:
 
-    - `genkit.config.ts`: this is where you configure Genkit for your project,
-      and select and configure the plugins you want to load. The sample config
-      loads a plugin to support the model provider you chose earlier.
+    - `index.ts`: this is your project's entry point, where you configure Genkit for your project,
```

Comment:
"... a sample source file, index.ts. This is your..."

(and fix indentation after)

Reply:
Modified; thanks for the feedback!